citrination_client/base/errors.py

```diff
@@ -0,0 +1,37 @@
+class CitrinationClientError(Exception):
```

Comment:
For all of these exceptions, would it make sense to be able to include the message received from the server?  It could be an optional argument into the constructor and appended to the message with something along the lines of "Received message from Citrination: " + server message.

Reply:
👍